src/common/attachments/uploadAttachmentsDialog.component.tsx

```diff
@@ -0,0 +1,177 @@
+import {
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  useTheme,
+} from '@mui/material';
+import AwsS3, { AwsBody } from '@uppy/aws-s3';
+import Uppy, { Meta, UppyFile } from '@uppy/core';
+import '@uppy/core/dist/style.css';
+import '@uppy/dashboard/dist/style.css';
+import ProgressBar from '@uppy/progress-bar'; // Import the ProgressBar plugin
+import { Dashboard } from '@uppy/react';
+import React from 'react';
+import { usePostAttachmentMetadata } from '../../api/attachments';
+
+const MAX_FILE_SIZE_MB = 100;
+const MAX_FILE_SIZE_B = MAX_FILE_SIZE_MB * 1000 * 1000;
```

Comment:
```suggestion
// Note: File systems use a factor of 1024 for GB, MB and KB instead of 1000, so here the former is expected despite them really being GiB, MiB and KiB.
const MAX_FILE_SIZE_MB = 100;
const MAX_FILE_SIZE_B = MAX_FILE_SIZE_MB * 1024 * 1024;
```
Looked into it more while adjusting the backend - I am using this comment to try and make it clearer.